llvm/CMakeLists.txt

```diff
@@ -289,6 +289,7 @@ set(LLVM_ALL_TARGETS
   AArch64
   AMDGPU
   ARM
+  AVR
```

Comment:
I'd drop this, to keep target lists in sync with LLVM 10.

Reply:
Good call, and rust-lang/master currently assumes AVR is an experimental backend so it is consistent there too.

I have dropped that commit (specifically cd16bb3c821) from the PR.